CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+- Fix an error for `RSpec/ReceiveMessages` with splat, symbol method. ([@marocchino])
```

Comment:
```suggestion
- Fix an incorrect autocorrect for `RSpec/ReceiveMessages` when method is only non-word character. ([@marocchino])
- Fix a false positive for `RSpec/ReceiveMessages` when return with splat. ([@marocchino])
```

Reply:
This PR addresses two issues, and since they are separated by commits, it would be nice if the CHANGELOG was updated for each commit.